test/Kestrel.Core.Tests/KestrelServerLimitsTests.cs

```diff
@@ -331,12 +331,27 @@ public void Http2HeaderTableSizeDefault()
         }
 
         [Theory]
-        [InlineData(4097)]
+        [InlineData(int.MinValue)]
         [InlineData(-1)]
         public void Http2HeaderTableSizeInvalid(int value)
         {
-            var ex = Assert.Throws<ArgumentOutOfRangeException>(() => new KestrelServerLimits().Http2.MaxFrameSize = value);
```

Comment:
I see the old test was improperly named, but we shouldn't remove coverage for MaxFrameSize if this isn't tested elsewhere. The old InlineData values should still be invalid for MaxFrameSize.

Reply:
I see. You might as well fix the inline data for the Http2MaxFrameSizeInvalid test. `1 << 14 - 1` should be `(1 << 14) - 1`.